.github/workflows/actions.yml

```diff
@@ -23,12 +24,15 @@ jobs:
           fetch-depth:        50
           submodules:         'recursive'
 
+      - name:                 Setup - Xcode
+        run:                  sudo xcode-select -switch '/Applications/Xcode_13.4.1.app/Contents/Developer' && /usr/bin/xcodebuild -version
```

Comment:
would we need to edit this line with every Xcode release?

Reply:
No, we only need to update this once this XCode version won't be able to properly build app version or one of Homebrew dependencies will require XCode version bump.
So far default for Github Action env is 13.2, but `swiftgen` requires `13.3` hence manual bump to newest version 🙏🏻